examples/tools/spec_puppet/content/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
You'll need to set execute permissions on this script before building the container, otherwise you'll get an error